### ISSUE TITLE
Fedora?

### ISSUE THREAD

**Issue by loctauxphilippe** [LeoIannacone/goopg#15]
Do you have plans to build a Fedora pacakge for this awesome tool?
I'd like to use it but I'm on Fedora...

**Comment by LeoIannacone** [comment]
Hi,

this should be absolutely addressed.

If you know somebody expert in Fedora packaging, please consider to ask for help.

**Comment by loctauxphilippe** [comment]
@LeoIannacone I know someone that can package for Fedora, but he's not an expert, me!

**Comment by LeoIannacone** [comment]
Then consider a PR :)

**Comment by loctauxphilippe** [comment]
@LeoIannacone I have a bunch of work to do, but I'll try to do it when I'll have time! ;-)

**Comment by LeoIannacone** [comment]
In the meanwhile you can proceed reading the [dev-install](../blob/master/doc/dev-install.md)

**Comment by travisby** [comment]
Hi -- trying the same thing on CentOS here :).

I'm having problems understanding the difference of the two pieces here, and which is the one that is being installed via apt in debian/ubuntu.  What part of the source is dedicated to that, and which is the chrome extension?

I'll see if I can get a package made up and thrown into EPEL (but I'm not quite sure what I'm trying to build! Heh.  The link above sounds like the extension, but I could be mistaken).

If you could either write something up quick, or give me a small architecture explanation between how the extension communicates with the... "plugin" I think you call it?  That'd be great.


Thanks for the guidance :D

**Comment by LeoIannacone** [comment]
Hi @travisby !

Good news!

In a nutshell:
The `host` directory contains the PLUGIN (python code) which must be installed via the package manager.
The `app` directory contains the EXTENSIONS which must be installed via the chrome extension webstore.

The EXTENSIONS "watches" at Gmail and sends info to the PLUGIN (via [Native Messaging](https://developer.chrome.com/extensions/nativeMessaging)). 
The PLUGIN reveices this info and uses the [Gmail API](https://developers.google.com/gmail/api/) to retreive the original message and then uses [python-gnupg](https://bitbucket.org/vinay.sajip/python-gnupg) to perform GPG signature and verification.

So, in other words, what you have to do is install the `host` directory correctly.

I strongly recommend to give a look at the [debian](https://github.com/LeoIannacone/goopg/tree/master/debian) directory:
 * the `rules` files builds the JSON file
 * the `*.install` files move files correctly to the right paths

Let me know if you need any more help..
